lib/PhpParser/Builder/ClassConst.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpParser\Builder;
+
+use PhpParser;
+use PhpParser\BuilderHelpers;
+use PhpParser\Node\Const_;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Stmt;
+
+class ClassConst implements PhpParser\Builder
+{
+    protected $flags = 0;
+    protected $attributes = [];
+    protected $constants = [];
+
+    /**
+     * Creates a class constant builder
+     *
+     * @param string|Identifier                          $name  - Name
+     * @param Node\Expr|bool|null|int|float|string|array $value — Value
```

Comment:
```suggestion
     * @param string|Identifier                          $name  Name
     * @param Node\Expr|bool|null|int|float|string|array $value Value
```
These dashes shouldn't be there (same below).